index.html

```diff
@@ -235,4 +235,32 @@ <h1>About<br>Myself</h1>
         <img src="./style/img/Separator Bottom.png" alt="separater">   
          
     </div>
+</section>
+
+<!-- Contact me Form-->
+<section class="contact">
+    <div class="contact-container">
+        <div class="contact-block">
+            <h2>Contact me</h2>
+        </div>
+        <div class="contact-text">
+            <p class="para">If you have an application you are interested in developing,
+                a feature that you need built or a project that needs coding. 
+                I’d love to help with it 
+            </p>
+        </div>
+<form action="https://formspree.io/f/mayklwek" method="POST">
+    <label for="user-name">
+        <input type="text" name="user-name" placeholder="Yeremias">
+    </label>
+    <label for="email">
+        <input type="email" name="email" placeholder="natajayanj14@gmail.com">
+    </label>
+    <label for="msg">
+        <textarea name="msg" placeholder="Write your messege" id="message"></textarea>
```

Comment:
Kindly make sure you are adding the validations required :+1: :smiley: :smiley: These are projects requirements.
- Name input should accept up to 30 chars.
- Textarea should accept up to 500 chars.
- A user cannot submit the form if one or more of the fields is empty.
 
## Check the image below, please.
![image](https://user-images.githubusercontent.com/98363075/199247134-b74bd5a7-7d91-4bfd-87d3-6743b3adb841.png)